src/index.js

```diff
@@ -6,7 +6,18 @@ import {OpenForm} from './sdk';
 const {REACT_APP_BASE_API_URL, REACT_APP_FORM_ID, REACT_APP_USE_HASH_ROUTING} = process.env;
 
 window.onload = () => {
-  const formId = new URLSearchParams(document.location.search).get('form');
+  const useHashRouting = REACT_APP_USE_HASH_ROUTING === 'true' || false;
+  const searchParams = new URLSearchParams(document.location.search);
+  const redirectPath = searchParams.get('redirect_path');
+  if (redirectPath) {
+    searchParams.delete('redirect_path');
+    let basePathname = document.location.pathname;
+    if (!basePathname.endsWith('/')) basePathname = `${basePathname}/`;
+    const pathname = `${basePathname}${useHashRouting ? '#/' : ''}${redirectPath}?${searchParams}`;
+
+    window.location.replace(new URL(pathname, document.location.origin));
+  }
```

Comment:
This is dirty and should either be moved in a helper function, or (even better):
- have the router created on `OpenForm.render` set as an attribute (and created on `init` instead of `render` so that we don't have to render the form and then perform a redirect).
- Use this router attribute here to programmatically redirect to the correct path

I hope this is doable?